http-core/src/main/scala/org/apache/pekko/http/javadsl/ServerBuilder.scala

```diff
@@ -174,27 +174,27 @@ object ServerBuilder {
 
     def bind(handler: Function[HttpRequest, CompletionStage[HttpResponse]]): CompletionStage[ServerBinding] =
       http.bindAndHandleAsyncImpl(
-        handler.apply(_).asScala,
+        handler.apply(_).asScala.map(_.asScala),
```

Comment:
The `.asScala` method here was conflict with the `org.apache.pekko.http.impl.util.AsScala.asScala` so I just did the conversion here directly.